tests/integrationtest/r/planner/core/issuetest/planner_issue.result

```diff
@@ -111,11 +111,12 @@ create table tbl_39(col_239 year(4) not null default '2009', primary key(col_239
 insert into tbl_39 values (1994),(1995),(1996),(1997);
 explain select  /*+ use_index_merge( tbl_39) */ col_239  from tbl_39 where not( tbl_39.col_239 not in ( '1994' ) ) and tbl_39.col_239 not in ( '2004' , '2010' , '2010' ) or not( tbl_39.col_239 <= '1996' ) and not( tbl_39.col_239 between '2026' and '2011' ) order by tbl_39.col_239 limit 382;
 id	estRows	task	access object	operator info
-Limit_12	382.00	root		offset:0, count:382
-└─IndexReader_39	382.00	root		index:Limit_38
-  └─Limit_38	382.00	cop[tikv]		offset:0, count:382
-    └─Selection_33	382.00	cop[tikv]		or(and(eq(planner__core__issuetest__planner_issue.tbl_39.col_239, 1994), not(in(planner__core__issuetest__planner_issue.tbl_39.col_239, 2004, 2010, 2010))), and(gt(planner__core__issuetest__planner_issue.tbl_39.col_239, 1996), or(lt(cast(planner__core__issuetest__planner_issue.tbl_39.col_239, double UNSIGNED BINARY), 2026), gt(cast(planner__core__issuetest__planner_issue.tbl_39.col_239, double UNSIGNED BINARY), 2011))))
-      └─IndexRangeScan_32	477.50	cop[tikv]	table:tbl_39, index:PRIMARY(col_239)	range:[1994,1994], (1996,+inf], keep order:true, stats:pseudo
+Projection_8	382.00	root		planner__core__issuetest__planner_issue.tbl_39.col_239
```

Comment:
The result before and after are both correct, it seems the 'tidb_partition_prune_mode' differ.